public/app/features/scopes/dashboards/ScopesNavigationTreeLink.tsx

```diff
@@ -0,0 +1,67 @@
+import { css } from '@emotion/css';
+import { useMemo } from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+import { GrafanaTheme2, IconName, locationUtil } from '@grafana/data';
+import { Icon, useStyles2 } from '@grafana/ui';
+
+export interface ScopesNavigationTreeLinkProps {
+  to: string;
+  title: string;
+}
+
+export function ScopesNavigationTreeLink({ to, title }: ScopesNavigationTreeLinkProps) {
+  const styles = useStyles2(getStyles);
+  const linkIcon = useMemo(() => getLinkIcon(to), [to]);
+
+  return (
+    <Link to={to} className={styles.container} data-testid={`scopes-dashboards-${title}`} role="treeitem">
```

Comment:
Does this one take subpaths into account?

Reply:
What do you mean by "subpaths"?

Reply:
If you serve Grafana from a subpath. `www.mydomain.com/grafana/a/grafana-my-app`.